docs/config.md

```diff
@@ -352,7 +365,18 @@ FSI
 }
 ```
 
-This will force the adapter use the boundary condition implementations
+Note that the adapter [does not currently adapt the name of the pressure used in the computations for compressible solvers](https://github.com/precice/openfoam-adapter/issues/253).
```

Comment:
```suggestion
Note that the adapter [does not adapt the name of the pressure used in the computations for compressible solvers](https://github.com/precice/openfoam-adapter/issues/253).
```

Reply:
But we want to fix that, eventually, no?

Reply:
Yes, the currently just seemed a bit odd in the sentence.